.gitignore

```diff
@@ -77,6 +77,7 @@ doc/generated/
 target/
 
 # VSCode
+.venv
```

Comment:
Please remove the environment from the MAPIE folder!
Thank you

Reply:
done